scripts/events.py

```diff
@@ -628,7 +628,7 @@ def one_moon_outside_cat(self, cat):
         if cat.moons == 1:
             cat.age = "kitten"
             if cat.status not in [
-                'kittypet', 'loner', 'rogue', 'former clancat'
+                'kittypet', 'loner', 'rogue', 'former Clancat'
```

Comment:
Is 'former Clancat' a valid status?
I thought 'exiled' is used for ex-Clancats.